solutions/security/get-started/enable-threat-intelligence-integrations.md

```diff
@@ -19,4 +19,78 @@ $$$agent-ti-integration$$$
 
 $$$custom-ti-integration$$$
 
-$$$ti-mod-integration$$$
\ No newline at end of file
+$$$ti-mod-integration$$$
+
+
+The Threat Intelligence view provides a streamlined way to collect threat intelligence data that you can use for threat detection and matching. Threat intelligence data consists of  [threat indicators](../../../troubleshoot/security/indicators-of-compromise.md#ti-indicators) ingested from third-party threat intelligence sources.
+
+Threat indicators describe potential threats, unusual behavior, or malicious activity on a network or in an environment. They are commonly used in indicator match rules to detect and match known threats. When an indicator match rule generates an alert, it includes information about the matched threat indicator.
+
+::::{note}
+To learn more about alerts with threat intelligence, visit [View alert details](../../../solutions/security/detect-and-alert/view-detection-alert-details.md).
+
+::::
+
+
+You can connect to threat intelligence sources using an [{{agent}} integration](../../../solutions/security/get-started/enable-threat-intelligence-integrations.md#agent-ti-integration), the [Threat Intel module](../../../solutions/security/get-started/enable-threat-intelligence-integrations.md#ti-mod-integration), or a [custom integration](../../../solutions/security/get-started/enable-threat-intelligence-integrations.md#custom-ti-integration).
+
+:::{image} ../../../images/serverless--getting-started-threat-intelligence-view.png
```

Comment:
Need to remove this dash and potentially the `serverless` component of this image file name.

Reply:
```suggestion
:::{image} ../../../images/getting-started-threat-intelligence-view.png
```